web/packages/extension/manifest.json5

```diff
@@ -22,7 +22,7 @@
             "run_at": "document_start",
         }
     ],
-    "content_security_policy": "default-src 'self'; script-src 'self' 'unsafe-eval'; style-src 'unsafe-inline'; connect-src *; img-src data:;",
+    "content_security_policy": "default-src 'self'; script-src 'self'; style-src 'unsafe-inline'; connect-src *; img-src data:;",
```

Comment:
Maybe add a comment here explaining that Webpack modifies this field?
(comments are a feature in JSON5)